### ISSUE TITLE
iPhone 6 not loading the wallet

### ISSUE THREAD

**Issue by BvL13** [swarmcity/sc-boardwalk#192]
Actual behavior:
After entering the password, the unlocking page shows and the wheel keeps on turning
(I can open the account on my desktop computer with no problem)

Expected behavior:
Wallet should open

Steps to reproduce the problem:
Open https://swarm.city
"Enter password" screen shows
Close screen to copy the password from another app
Go back to enter password screen
Paste Password
Unlock screen shows but nothing happens

**Comment by bkawk** [comment]
Can we assume this is in Safari? If it is we can debug this using the opera dev tools https://developer.apple.com/library/content/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/GettingStarted/GettingStarted.html

**Comment by sophiii** [comment]
Can you test this with chrome on iphone?

**Comment by BvL13** [comment]
I tested with chrome and it is the same behavior. Therefore it must an iPhone issue and not a browser issue.

**Comment by BvL13** [comment]
One more issue I experienced in using chrome on iPhone:
In chrome I go through the full Token swap process now. Thats new to me.
I pressed "No, dont have any" 
Then I get stuck when I need to enter my password, but I actually want to reopen my existing account.
Only if I press the x, then I get to the "enter here screen"

**Comment by BvL13** [comment]
Background info:
I did not have chrome installed before on my iphone.

**Comment by sponnet** [comment]
@BvL13 : can you confirm that this issue is still present after the release of boardwalk ?

**Comment by BvL13** [comment]
Yes, I still cannot access swarm city. When I try to enter the password the screen either freezes (no activity is possible) or it is blank (white screen)
I am using safari.
